test/integration/helpers_test.go

```diff
@@ -83,11 +83,6 @@ func DeployGateway(ctx context.Context, client *gatewayclient.Clientset, namespa
 					Protocol: gatewayv1beta1.HTTPProtocolType,
 					Port:     gatewayv1beta1.PortNumber(ktfkong.DefaultProxyTCPServicePort),
 				},
-				{
-					Name:     "https",
-					Protocol: gatewayv1beta1.HTTPSProtocolType,
-					Port:     gatewayv1beta1.PortNumber(ktfkong.DefaultProxyTLSServicePort),
-				},
```

Comment:
What's the reason for this removal?

Reply:
HTTPS Listeners now require a TLS configuration. Adding one here was awkward and would either require a generic cert or signature change, and most test cases don't actually make use of it.

The one test that was (GRPC) now adds its own Listener using `opts`, following the same pattern as other tests that need TLS or HTTPS.